app/pb_kits/playbook/pb_selectable_card/selectable_card.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbSelectableCard
+    class SelectableCard
+      include Playbook::Props
+
+      partial "pb_selectable_card/selectable_card"
+
+      prop :checked, type: Playbook::Props::Boolean,
+                  default: false
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :disabled, type: Playbook::Props::Boolean,
+                  default: false
+      prop :icon, type: Playbook::Props::Boolean,
+                  default: true
+      prop :multi, type: Playbook::Props::Boolean,
+                  default: true
+      prop :name
+      prop :text
+      prop :value
+
+      def classname
+        generate_classname("pb_selectable_card_kit", dark_class, disabled_class)
+      end
+
+      def html_for
+        id.present? ? id : name
+      end
+
+      private
+
+      def dark_class
+        dark ?  "dark" : nil
+      end
+
+      def disabled_class
+        disabled ? "disabled" : "enabled"
```

Comment:
This is a little odd to read. To me it says:

> The `disabled_class` method will return "disabled" if `disabled`, but if NOT `disabled` will return "enabled"

Can we just default to "enabled" and change this method to:

```suggestion
        "disabled" if disabled
```